build/preprocessing.py

```diff
@@ -0,0 +1,72 @@
+#! python
+
+import yaml
+import sys
+from os import listdir
+from os.path import isfile, join
+
+if __name__ == "__main__":
+
+  print("Running preprocessing for common templates")
+
+  rules = None
+  patchPath = sys.argv[1] if len(sys.argv) > 1 else "patch.yaml"
+  with open(patchPath, 'r') as stream:
+    try:
+      patch = yaml.load(stream, Loader=yaml.BaseLoader)
+
+      if patch == None: 
+        print("Nothing in patch file")
+        sys.exit(0)
+      
+      rules = patch.get("rules")
+      for rule in rules:
+        rule["matchLabel"] = rule.get("matchLabel").split(": ")
+        rule["addAnnotation"] = rule.get("addAnnotation").split(": ")
+    except yaml.YAMLError as exc:
+      print(exc)
+      sys.exit(1)
+
+  commonTemplatesPath = sys.argv[2] if len(sys.argv) > 2 else "/opt/ansible/roles/KubevirtCommonTemplatesBundle/files"
```

Comment:
please use constants (well, actually module-level globals) to make evident the defaults we use.

Reply:
Also, do we want to use something like the argparse module?

Reply:
I do not expect any other arguments so it might be a bit of an overkill.